src/routes/+layout.svelte

```diff
@@ -214,7 +214,6 @@
                             placement: "bottom-end",
                             offset: [0, 10],
                             animation: "shift-away",
-                            appendTo: document.body,
```

Comment:
If `interactive: true`, the default behavior is `appendTo: "parent"` instead of `document.body`.

LOL do we need to append to document?

Reply:
Nooo idk. Do what you think is best

Reply:
>No : Cause it doesn't make sense for popover to be appended to parent. Styling issue will happen
Yes : Cause we can get style isolation

Yea, now I think we can go with appending to document.
Maybe skeleton cause style issues when we append to parent ...